src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,41 @@
+package org.example;
+
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class Main
+{
+    private static final Logger log = Logger.getLogger(Main.class);
+    private static final Logger orderLogger = Logger.getLogger("SECOND_LOGGER");
```

Comment:
It's kind of hack to write file with a logger 😄 
+1 for respect for such approach, but let's use general approach)